### PR TITLE
HTML STRUCTURE

### PR DESCRIPTION
**Pull request by hizikex**
WHY THIS PR?
WHAT DOES THIS PR DO?
WHAT SHOULD REVIEWERS KNOW?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+
```
Comment (henrytrust1111) [inline index.html:9]: Empty body

File: index.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
```
Comment (DevEben) [inline index.html:8]: There is nothing body your code

Comment (DevRajah) [inline index.html:8]: Boss, nothing dey your code

File: index.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    
+</body>
+</html>
```
Comment (Femoxy) [inline index.html:11]: There should be more to this code

**Review by Ekele1 — COMMENTED** [review]
empty body

File: index.html (hunk 2 above)
Comment (hizikex) [reply index.html:8]: leave it like that

File: index.html (hunk 1 above)
Comment (Godwin024) [reply index.html:9]: change that thing!

File: index.html (hunk 2 above)
Comment (DevRajah) [inline index.html:8]: nothing

**Review by Femoxy — APPROVED** [review]

**Review by DevEben — APPROVED** [review]

**Comment by AKBOIY** [comment]
not completed

**Review by Ekele1 — APPROVED** [review]
empty body

**Review by henrytrust1111 — APPROVED** [review]

**Pull request merged**